### DISCUSSION TITLE
Disqus Commenting System

### DISCUSSION THREAD

**Discussion by pastocaza** [timlrx/tailwind-nextjs-starter-blog#101] [Ideas]
Hi @timlrx,

First of all, thank you so much for this template. Could we have an option for implementing Disqus in the posts?

I already did it in my blog, and it's pretty simple and works well, except for the theme in Dark Mode for Chromium desktop builds (works fine in firefox or mobile).

I don't know if it's a problem with Tailwind or Next, most likely is Tailwind and the way they handle changing dark/light model in the `root` itself and not in the `body`.

So I can probably PR that change and then try to fix the issue? I don't know how many people would like to add comments to their blog 😄 

Thank you!

**Comment by salmanhk** [comment]
A commenting system would be great. Would it be possible to have a switchable mechanism in case people want alternatives like Commento (i'm not affiliated). From what I understand Disqus has a heavier footprint compared to alternatives and privacy issues as well.

**Comment by timlrx** [comment]
Yes, see [related issue](https://github.com/timlrx/tailwind-nextjs-starter-blog/issues/111). @Music47ell just contributed a [PR](https://github.com/timlrx/tailwind-nextjs-starter-blog/pull/115) which adds support for Disqus, utterances and giscus!
